docs/volunteering/calendars.md

```diff
@@ -4,7 +4,15 @@ We use a few calendars to organise events, both public and private.
 
 ## Volunteer Calendar
 
-We use [a Google calendar](https://calendar.google.com/calendar/embed?src=studentrobotics.org_oqdjasvpps8smo0d5nte417rak%40group.calendar.google.com&ctz=Europe%2FLondon) to track events relevant to volunteers.
+We use [a Google calendar](https://calendar.google.com/calendar/embed?src=studentrobotics.org_oqdjasvpps8smo0d5nte417rak%40group.calendar.google.com&ctz=Europe%2FLondon) to track events relevant to volunteers. It is implied that if the event is in this shared calendar, any volunteer is welcome to attend.
```

Comment:
Could you clarify the purpose of the final sentence here:

> It is implied that if the event is in this shared calendar, any volunteer is welcome to attend.

I think I'd rather encourage all events to be listed in a single place rather than encourage lots of separate calendars (seeing what other teams do both builds empathy and a feel for whether you might want to join). Pairing that with "if you've not worked with a team before, reach out to the organisers before turning up to check whether they'll be able to help onboard you" feels like it should avoid confusing/accidental time wasting. What do you think?

Reply:
It makes it much more obvious that anyone is welcome, especially given that anyone who can see the calendar can see the meeting links.

I would assume that any team meeting falls under this "Any volunteer is welcome" banner.